modules/stereo/include/opencv2/stereo/quasi_dense_stereo.hpp

```diff
@@ -0,0 +1,234 @@
+/*
+By downloading, copying, installing or using the software you agree to this license.
+If you do not agree to this license, do not download, install,
```

Comment:
Please use short OpenCV license header:

```
// This file is part of OpenCV project.
// It is subject to the license terms in the LICENSE file found in the top-level directory
// of this distribution and at http://opencv.org/license.html.
```

https://github.com/opencv/opencv/wiki/Coding_Style_Guide

You may add "author" information after these 3 lines.